src/OpenTelemetry.Extensions.PersistentStorage/FileBlobProvider.cs

```diff
@@ -62,6 +62,27 @@ public class FileBlobProvider : PersistentBlobProvider, IDisposable
     /// Controls the timeout when writing a buffer to blob.
     /// Default is 1 minute.
     /// </param>
+    /// <exception cref="ArgumentNullException">
+    /// path is null.
```

Comment:
Consider some best practices like https://www.pluralsight.com/guides/using-param-paramref-tag-in-csharp-xml-documentation-comments.

Reply:
will follow up separately.